### PR TITLE
log: colorize stderr output in tmux and screen.

### PR DESCRIPTION
**Pull request by knz**
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12652)

### CODE REVIEW

File: pkg/util/log/log_scope_test.go [hunk 1]
```diff
@@ -1,115 +0,0 @@
-// Copyright 2016 The Cockroach Authors.
```
Comment (mberhault) [inline pkg/util/log/log_scope_test.go]: is this file deletion intentional?

Comment (knz) [inline pkg/util/log/log_scope_test.go]: wow no.

.. and that's why reviews are useful!

PTAL

**Review by a-robinson — APPROVED** [review]
LGTM

**Comment by knz** [comment]
TFYR :smile:

**Pull request merged**